azerothcore/azerothcore-wotlk
# fix(DB/GO): prevents four objects from spawning in the Stormwind Moon…

…well



## Changes Proposed:
-  Prevents the four objects (ID = 177272 - GUID = 42907, 42908, 42909, 42910) from spawning in the pool.

## Issues Addressed:

- Closes #5817

## SOURCE:

Before:
![image](https://user-images.githubusercontent.com/53914190/126017341-ee4f785d-fbae-4c5e-b240-164fd5b85382.png)

After:
![image](https://user-images.githubusercontent.com/53914190/126017342-7b77d428-ac98-4ede-b077-e23f6977c4e2.png)


## Tests Performed:

- Checked that the SQL command changes four rows
- .go o 42907
- Observe


## How to Test the Changes:


1. Check that the SQL command changes four rows
2. .go o 42907
3. Observe (Should say that there is no such object)

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
